modules/adding-a-data-connection-to-your-data-science-project.adoc

```diff
@@ -28,11 +28,11 @@ A project details page opens.
 +
 The *Add data connection* dialog opens.
 . Enter a *name* for the data connection.
-. Enter your access key ID for Amazon Web Services in the *AWS_ACCESS_KEY_ID* field.
-. Enter your secret access key for the account you specified in the *AWS_SECRET_ACCESS_KEY_ID* field.
-. Enter the endpoint of your AWS S3 storage in the *AWS_S3_ENDPOINT* field.
-. Enter the default region of your AWS account in the *AWS_DEFAULT_REGION* field.
-. Enter the name of the AWS S3 bucket in the *AWS_S3_BUCKET* field.
+. Enter your access key ID for your chosen S3-compatible object storage provider in the *Access key* field.
```

Comment:
A couple of minor suggestions here:

- I prefer "Enter **the** access key ID"
- I think we can eliminate "chosen". It might give users the impression that they were supposed to choose or select a provider somewhere, which is not the case.

Reply:
Also, let's flip all of these instructions to put the condition/location of the action first:

1. In the **Access key** field, enter....
2. In the **Secret key** field, enter...

Reply:
Addressed comments, preview below:

![Screenshot 2023-11-22 at 9 17 42 AM](https://github.com/opendatahub-io/opendatahub-documentation/assets/149716269/d10f1bb7-e5ea-4ddb-aecf-f873b1f8a163)